docs/users/getting-started/configuration.md

```diff
@@ -65,15 +60,15 @@ Settings that affect all editors used inside BetterDiscord.
 
 ### Line Numbers
 
-This option simply determines whether or not line numbers should be shown.
+This option determines whether or not line numbers should be shown.
```

Comment:
why

Reply:
the word simply isn't really necessary but if you don't want to change it then its fine.

Reply:
since I want to just resolve this I have decided to revert back to the old text